src/main/java/gameboy/gamepack/racinggame/view/InputView.java

```diff
@@ -1,25 +1,73 @@
 package gameboy.gamepack.racinggame.view;
 
-import java.util.InputMismatchException;
-import java.util.Scanner;
+import java.util.*;
+
+import gameboy.gamepack.racinggame.data.vo.Name;
+import gameboy.gamepack.racinggame.exception.InvalidRacerNameException;
 
 public class InputView {
 
-    public static int inputNumber(String ask) {
+    private static final int MAX_NAME_SIZE = 5;
+    private static final String CARS_NAME_SEPARATOR = ",";
+
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static int inputRaceCount() {
+        return inputNumber("시도할 회수는 몇 회 인가요?");
+    }
+
+    public static List<Name> inputCarsName() {
+        return inputCarsName("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).").stream().map(Name::new).toList();
+    }
+
+    private static List<String> inputCarsName(String ask) {
+        try {
+            List<String> carsName = split(inputText(ask), CARS_NAME_SEPARATOR);
+            carsName.forEach(InputView::validRacerName);
+            return carsName;
+        } catch (InvalidRacerNameException e) {
+            return inputCarsName(e.getMessage());
+        }
+    }
+
+    private static int inputNumber(String ask) {
         try {
             display(ask);
             return userIntInput();
-        }catch (InputMismatchException e) {
+        } catch (InputMismatchException e) {
             return inputNumber("숫자만 입력해주세요.");
         }
     }
 
-    private static void display(String ask) {
-        System.out.println(ask);
+    public static String inputText(String ask) {
+        display(ask);
+        return userTextInput();
+    }
+
+    private static void display(String message) {
+        System.out.println(message);
     }
 
     private static int userIntInput() {
-        Scanner scanner = new Scanner(System.in);
         return scanner.nextInt();
     }
+
+    private static String userTextInput() {
+        return scanner.nextLine();
+    }
+
+    private static List<String> split(String text, String separator) {
+        List<String> tokens = List.of(text.split(separator));
+        return tokens.stream().map(String::trim).toList();
+    }
```

Comment:
좀 더 간결하게 표현할 수 있겠네요
```suggestion
    private static List<String> split(String text) {
        return Stream.of(text.split(CARS_NAME_SEPARATOR)).map(String::trim).toList();
    }
```

separator를 파라미터로 받아서 처리하는 것도 좋아보이는데
메소드가 public일 때 한정입니다.
지금은 private 내부적으로 사용하고 있기 때문에 내부 고정값 상수 `CARS_NAME_SEPARATOR`를 사용하는 게 맞습니다.

Reply:
split 메소드가 inputCarsName에서 사용하는데 메서드 거리가 너무 멀어요
이것 말고도 InputView에 있는 메소드 전체를 보시고 배치를 고려해보시면 좋겠습니다.